cli/smartcontract/smart_contract.go

```diff
@@ -224,10 +226,10 @@ func NewCommands() []cli.Command {
     * '03b209fd4f53a7170ea4444e0cb0a6bb6a53c2bd016926989cf85f9b0fba17a70c' is a
       key with a value of '03b209fd4f53a7170ea4444e0cb0a6bb6a53c2bd016926989cf85f9b0fba17a70c'
 
-   Cosigners represent a set of Uint160 hashes with witness scopes and are used
-   to verify hashes in System.Runtime.CheckWitness syscall. To specify cosigners
-   use cosigner[:scope] syntax where
-    * 'cosigner' is hex-encoded 160 bit (20 byte) LE value of cosigner's address,
+   Signers represent a set of Uint160 hashes with witness scopes and are used
+   to verify hashes in System.Runtime.CheckWitness syscall. To specify signers
+   use signer[:scope] syntax where
+    * 'signer' is hex-encoded 160 bit (20 byte) LE value of signer's address,
```

Comment:
It it LE or BE ?

Reply:
LE because it's external representation for users, as in C# node.